api/turing/api/appcontext.go

```diff
@@ -16,14 +16,12 @@ import (
 	"github.com/caraml-dev/turing/api/turing/cluster/labeller"
 	"github.com/caraml-dev/turing/api/turing/config"
 	"github.com/caraml-dev/turing/api/turing/imagebuilder"
-	"github.com/caraml-dev/turing/api/turing/middleware"
 	"github.com/caraml-dev/turing/api/turing/service"
 	"github.com/caraml-dev/turing/engines/router/missionctl/errors"
 )
 
 // AppContext stores the entities relating to the application's context
 type AppContext struct {
-	Authorizer *middleware.Authorizer
```

Comment:
This is removed because the Turing API server no longer uses the Authorization middleware layer.

Reply:
this was used before the Keto upgrade?

Reply:
Yes that's right.